jujugui/static/gui/src/app/components/inspector/test-inspector.js

```diff
@@ -36,19 +36,44 @@ describe('Inspector', function() {
       get: function() {
         return {name: 'demo'};
       }};
-    var getAppState = function() {
-        return 'inspector';
-      };
+    var appState = {
+      sectionA: {
+        metadata: {}
+      }};
+    var shallowRenderer = testUtils.createRenderer();
```

Comment:
Yeah I actually wrote that wrapper while at the very end of this branch heh so a bunch weren't updated. I'll have to go back and update them.